kratos/python/add_strategies_to_python.cpp

```diff
@@ -221,8 +222,8 @@ namespace Kratos
 
             binder.def("ClearMatrix", ClearMatrix<TSpaceType>);
             binder.def("ClearVector", ClearVector<TSpaceType>);
-            binder.def("ResizeMatrix", ResizeMatrix<TSpaceType>);
-            binder.def("ResizeVector", ResizeVector<TSpaceType>);
+            // binder.def("ResizeMatrix", ResizeMatrix<TSpaceType>);
```

Comment:
Remove?

Reply:
i would prefer to keep it for a while, to optionally throw a meaningful error